website/docs/reference/network-view.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Network view
+---
+
+import Figure from '@site/src/components/Figure/Figure.tsx'
+
+:::info Availability
+
+The network view was released in Unleash 4.21. It is available to Pro and Enterprise users.
+
+You must configure the [data source](#data-source) to activate the feature.
+
+:::
+
+The Unleash admin UI contains a **network view** as part of the admin configuration pages. This network view was designed to give you an overview and understanding of incoming requests to your Unleash instance. It makes it possible to pinpoint sources of sudden request surges, and can therefore also help you identify issues with SDK setups[^1].
+
+The network view offers two different visualizations of the same data, known as the [network overview](#network-overview) and the [network traffic](#network-traffic) views.
+
+Because Unleash doesn't store this kind of data itself, the network view requires you to configure an [external data source](#data-source) for the network overview to work. The network view is only mad available if you tell Unleash where it can find the data (refer to the [data source section](#data-source)).
```

Comment:
```suggestion
Because Unleash doesn't store this kind of data itself, the network view requires you to configure an [external data source](#data-source) for the network overview to work. The network view is only available if you tell Unleash where it can find the data (refer to the [data source section](#data-source)).
```
I think this was "made available", but also it can just be removed

Reply:
Absolutely! Good catch.